src/client/gateways/Gateway.ts

```diff
@@ -5,7 +5,7 @@ import _ from 'lodash'
 type ApiData = object | string
 
 const THIRTY_SECONDS = 1000 * 30
-const ONE_MINUTES = 1000 * 60 * 60
+const ONE_MINUTES = 1000 * 60
```

Comment:
I felt it was a mistake, let me know if I have to revert it.

Reply:
Yeah looks like a mistake to me, copy and paste error most likely!